integrations/visual-website-optimizer/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@segment/analytics.js-integration-visual-website-optimizer",
+  "description": "The Visual Website Optimizer analytics.js integration.",
+  "version": "2.3.0",
+  "keywords": [
+    "analytics.js",
+    "analytics.js-integration",
+    "segment",
+    "visual-website-optimizer"
+  ],
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "karma start",
+    "test:ci": "karma start karma.conf-ci.js"
+  },
+  "author": "Segment <friends@segment.com>",
+  "license": "SEE LICENSE IN LICENSE",
+  "homepage": "https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/visual-website-optimizer#readme",
+  "bugs": {
+    "url": "https://github.com/segmentio/analytics.js-integrations/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/segmentio/analytics.js-integrations.git"
+  },
+  "dependencies": {
+    "@segment/analytics.js-integration": "^3.1.0",
+    "component-each": "^0.2.6",
+    "next-tick": "^0.2.2"
+  },
+  "devDependencies": {
```

Comment:
@CarlosMecha - I think it's useful to have integrations share the same core dev dependencies if possible (ie karma, mocha, etc). Also, many of these aren't actually required. Here's the updated dev deps in my earlier PR: https://github.com/segmentio/analytics.js-integrations/pull/189/files#diff-73c610cfbf55667abd8aa7f8c836c158R31-R46. The tests all pass fine with them so there should not be any issues.